sdk/include/opentelemetry/sdk/metrics/data/point_data.h

```diff
@@ -17,38 +17,62 @@ namespace metrics
 {
 
 using ValueType = nostd::variant<long, double>;
-using ListType  = nostd::variant<std::vector<long>, std::vector<double>>;
+using ListType  = nostd::variant<std::list<long>, std::list<double>>;
+
+// TODO: remove ctors and initializers from below classes when GCC<5 stops shipping on Ubuntu
+// refer error:
+// https://github.com/open-telemetry/opentelemetry-cpp/runs/5577740472?check_suite_focus=true#step:6:522
```

Comment:
This link will be valid only for 90 days I think.

Reply:
Yes, I think we can remove the link, as the details are mentioned in comment.